site-src/blog/2022/first-beta.md

```diff
@@ -0,0 +1,142 @@
+---
+description: >
+  We're pleased to announce the start of beta releases for the Gateway API
+  project with the release of version `v0.5.0`. This release includes `v1beta1`
+  versions of `GatewayClass`, `Gateway` and `HTTPRoute` as well as several other
+  significant improvements which this post will cover some of the highlights of.
+---
+
+# Introducing Gateway API v1beta1
+
+<small style="position:relative; top:-30px;">
+  :octicons-calendar-24: June 22, 2022 ·
+  :octicons-clock-24: 5 min read
+</small>
+
+We're pleased to announce the start of beta releases for the Gateway API
+project with the release of version `v0.5.0`. This release includes `v1beta1`
+versions of `GatewayClass`, `Gateway` and `HTTPRoute` as well as several other
+significant improvements which this post will cover some of the highlights of.
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around `Gateway` resources
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known `Ingress` API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `GRPC`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a native Kubernetes API (whereas previously these might
+have been managed somewhere else, like helm).
+
+If you're an end-user interested in some of the benefits of Gateway API we
+invite you to jump in and find an implementation that suits you. At the time of
+this release there are over a dozen implementations for popular API gateways and
+service meshes and guides are available to start exploring quickly.
+
+### Getting Started
```

Comment:
Somewhere in here I think we need to clarify that this API is built on CRDs. That means that anyone with a Kubernetes 1.16+ cluster can install the latest set of APIs and get up and running with it.

Reply:
I think "Getting Started" probably fits best after "Release Highlights" and before "Next Steps"

Reply:
Sounds good, added in 82421d13bd444b49980d66e8db97824026d5d053